crates/dwallet-mpc-types/src/dwallet_mpc.rs

```diff
@@ -31,6 +32,99 @@ pub const START_NETWORK_DKG_EVENT_STRUCT_NAME: &IdentStr =
 /// Alias for an MPC message.
 pub type MPCMessage = Vec<u8>;
 
+/// Represents a chunk of an MPC message, with metadata for reassembly.
+#[derive(Clone, Debug, Serialize, Deserialize, Hash, PartialEq, Eq, Ord, PartialOrd)]
+pub struct MPCMessageSlice {
+    /// A fragment of the original message.
+    pub message: MPCMessage,
+    /// The position of this chunk in the original message sequence.
+    pub sequence_number: u64,
+    /// Total number of chunks in the message.
+    pub number_of_chunks: usize,
```

Comment:
This is uselessly duplicated in each message. Should be once in a header or the first message